snmp/datadog_checks/snmp/data/profiles/_base_cisco.yaml

```
@@ -6,6 +6,9 @@ extends:
   - _generic-router-tcp.yaml
   - _generic-router-udp.yaml
 
+tags: 
+  "vendor:cisco"
```

```suggestion
metric_tags:
  - "vendor:cisco"
```

And same for the rest